articles/connections/database/custom-db/metadata.md

```diff
@@ -0,0 +1,9 @@
+---
+description: Learn how to work with metadata when using your database as an identity provider.
+crews: crew-2
+---
+# Metadata and Custom Databases
+
```

Comment:
what's this doc about? in which case should I read it?

how are the metadata relevant to the IP whitelisting that follows?